media/doc/3rd Party Files.txt

```diff
@@ -151,6 +151,12 @@ Used Version: ???
 License: GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later.html)
 URL: http://www.vmware.com/download/open_sources.html
 
+Title: AC97 Sample Driver
+Path: drivers/wdm/audio/drivers/AC97
```

Comment:
```suggestion
Path: drivers/wdm/audio/drivers/ac97
```

Reply:
I am no fan of frequent restructuring in the source tree.
Atm this folder does contain 3 subdirs: cmiDriver, mpu401, sb16. All of them are sound related. Fine.
I don't see any real benefit to move them around. I mean we are within root/drivers already. It's not like that they would be heavily misplaced within root/dll or something.

Reply:
Audio folders are a mess, `wdm` is a useless folder there. It needs more than just a renaming